src/Mod/RuleItem.h

```diff
@@ -635,6 +635,10 @@ class RuleItem
 	int getSpawnUnitFaction() const;
 	/// Check if LOS is required to use this item (only applies to psionic type items)
 	bool isLOSRequired() const;
+	/// Check if targeting friendly units is allowed.
```

Comment:
Function names and descriptions should also say this is for Psi only.

Reply:
Question, should not have similar functionality to medkit? AKA target self, friend, neutral, enemy, where enemy is only default. We could even add target empty. This could be reused in other context too (like melee).

Reply:
Well, I believe the current code would allow targeting neutrals when enemyTargeting is enabled. An empty/self-target seemed not needed, since, as far as I understand, melee works entirely different. But maybe this will become a moot point when is submit my next PR, which has an soldier skill menu, that explicitly allows melee and self-targeting actions.

Reply:
Point was to split neutral/enemy, with this you could have "Megaphone" that would allow control civilians during terror missions.